05/app.js

```diff
@@ -9,91 +9,149 @@ const voivodeship = document.getElementsByName("voivodeship");
 const form = document.querySelector("form");
 const ulElement = document.querySelector(".messages");
 
+var liElementFirstName = document.createElement("li");
+var liElementFirstNameUpper = document.createElement("li");
+var liElementLastName = document.createElement("li");
+var liElementLastNameUpper = document.createElement("li");
+var liElementStreet = document.createElement("li");
+var liElementStreetUpper = document.createElement("li");
+var liElementHouse = document.createElement("li");
+var liElementZip = document.createElement("li");
+var liElementCity = document.createElement("li");
+var liElementCityUpper = document.createElement("li");
+var liElementVoivodeship = document.createElement("li");
+
```

Comment:
ok, ale można było prościej.
Na samym początku masz `const errors = []`;
Do niego dodajesz błędy (to już miałeś).
Potem po każdym submit wyszukujesz tylko `ul` (.document.querySelector('ul'));
i robisz `ul.innerHTML = ''` - to wyczyści poprzednie błędy, a nowe pokażesz ;)